src/test/java/com/springbootpractice/examples/employeemanagement/controller/EmployeeServiceImplTest.java

```diff
@@ -0,0 +1,44 @@
+package com.springbootpractice.examples.employeemanagement.controller;
+
+import static org.mockito.Mockito.when;
+
+import java.util.Optional;
+
+import org.junit.Ignore;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.springbootpractice.examples.employeemanagement.dao.EmployeeRepository;
+import com.springbootpractice.examples.employeemanagement.data.Employee;
+import com.springbootpractice.examples.employeemanagement.service.EmployeeService;
+import com.springbootpractice.examples.employeemanagement.service.impl.EmployeeServiceImpl;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(EmployeeService.class)
+@Ignore
+public class EmployeeServiceImplTest {
```

Comment:
why 2 classed of the same name in 2 different directories

Reply:
I will remove this as it is not required anymore.